Lib/test/test_pdb.py

```diff
@@ -1486,6 +1486,14 @@ def test_relative_imports_on_plain_module(self):
         stdout, _ = self._run_pdb(['-m', self.module_name + '.runme'], commands)
         self.assertTrue(any("VAR from module" in l for l in stdout.splitlines()), stdout)
 
+    def test_syntaxerror(self):
+        stdout, _ = self.run_pdb_module("", "print(")
+        self.assertIn('(Pdb) *** SyntaxError: unexpected EOF while parsing\n(Pdb) ', stdout)
+
+    def test_syntaxerror_debug(self):
+        stdout, _ = self.run_pdb_module("", "debug print(")
+        self.assertIn('(Pdb) *** SyntaxError: unexpected EOF while parsing\n(Pdb) ', stdout)
```

Comment:
Would appreciate feedback on this.. should it be put into a single test?  (mainly wanting this to be fast)
As for this change only `test_syntaxerror_debug` could be added also.

Reply:
Also needs to be fixed for Windows to include `\r` / use `splitlines`.

Reply:
Hi, yes IMO would be better add whole in a single test, because you are testing the same things.

Reply:
> Also needs to be fixed for Windows to include `\r` / use `splitlines`.

Currently there are problems with Azure test

Reply:
> because you are testing the same things.

Not really.. the test for just `print(` was working before this patch already (and uses another code path).